--- foo2zjs/Makefile.bero	2013-01-15 19:10:38.386732166 +0100
+++ foo2zjs/Makefile	2013-01-15 19:11:06.077398457 +0100
@@ -888,7 +888,8 @@ install-hotplug-prog:
 
 install-filter:
 	if [ "$(CUPS_SERVERBIN)" != "" ]; then \
-	    ln -sf $(BIN)/command2foo2lava-pjl $(CUPS_SERVERBIN)/filter/; \
+	    mkdir -p $(DESTDIR)/$(CUPS_SERVERBIN)/filter/; \
+	    ln -sf ../../../../command2foo2lava-pjl $(DESTDIR)/$(CUPS_SERVERBIN)/filter/; \
 	fi
 
 cups:	FRC
