--- Makefile	2010-12-09 14:58:26.878875725 +0100
+++ Makefile.oden	2010-12-09 14:58:42.804900739 +0100
@@ -128,10 +128,6 @@
 		Makefile \
 		foo2zjs.c \
 		foo2zjs.1in \
-		jbig.c \
-		jbig.h \
-		jbig_ar.c \
-		jbig_ar.h \
 		zjsdecode.c \
 		zjsdecode.1in \
 		zjs.h \
@@ -259,7 +255,7 @@
 MANPAGES+=	gipddecode.1
 MANPAGES+=	foo2zjs-pstops.1 arm2hpdl.1 usb_printerid.1
 MANPAGES+=	printer-profile.1
-LIBJBG	=	jbig.o jbig_ar.o
+LIBJBG	=	-ljbig
 BINPROGS=
 
 ifeq ($(UNAME),Linux)
