diff -Naur foo2zjs/Makefile foo2zjs.tpg/Makefile
--- foo2zjs/Makefile	2015-04-29 06:27:27.000000000 +0000
+++ foo2zjs.tpg/Makefile	2015-04-29 06:28:53.301642789 +0000
@@ -127,10 +127,6 @@
 		Makefile \
 		foo2zjs.c \
 		foo2zjs.1in \
-		jbig.c \
-		jbig.h \
-		jbig_ar.c \
-		jbig_ar.h \
 		zjsdecode.c \
 		zjsdecode.1in \
 		zjs.h \
@@ -276,7 +272,7 @@
 MANPAGES+=	gipddecode.1
 MANPAGES+=	foo2zjs-pstops.1 arm2hpdl.1 usb_printerid.1
 MANPAGES+=	printer-profile.1
-LIBJBG	=	jbig.o jbig_ar.o
+LIBJBG	=	-ljbig
 BINPROGS=
 
 ifeq ($(UNAME),Linux)
