diff -Naurp foo2zjs/foo2hiperc-wrapper.1in foo2zjs.oden/foo2hiperc-wrapper.1in
--- foo2zjs/foo2hiperc-wrapper.1in	2009-01-21 04:24:53.000000000 +0100
+++ foo2zjs.oden/foo2hiperc-wrapper.1in	2010-12-09 15:02:22.012807086 +0100
@@ -144,7 +144,7 @@ etc.  This option may appear more than o
 .TP
 .BI \-G\0 profile.icm
 Convert \fIprofile.icm\fP to a Postscript color rendering
-dictionary (CRD) using \fBfoo2zjs-icc2ps\fP and
+dictionary (CRD) using \fBicc2ps\fP and
 adjust the printer colors by using the Postscript \fBsetcolorrendering\fP
 operator.  (WORK IN PROGRESS).
 .TP
diff -Naurp foo2zjs/foo2hiperc-wrapper.in foo2zjs.oden/foo2hiperc-wrapper.in
--- foo2zjs/foo2hiperc-wrapper.in	2010-11-12 13:13:40.000000000 +0100
+++ foo2zjs.oden/foo2hiperc-wrapper.in	2010-12-09 15:01:53.415806989 +0100
@@ -491,7 +491,7 @@ create_crd() {
     #
     # Create a Postscript CRD
     #
-    ICC2PS=$PREFIX/bin/foo2zjs-icc2ps
+    ICC2PS=$PREFIX/bin/icc2ps
     if [ -x $ICC2PS ]; then
 	case "$GAMMAFILE" in
 	none | none.icm | */none.icm)
diff -Naurp foo2zjs/foo2hp2600-wrapper.1in foo2zjs.oden/foo2hp2600-wrapper.1in
--- foo2zjs/foo2hp2600-wrapper.1in	2009-09-23 02:46:33.000000000 +0200
+++ foo2zjs.oden/foo2hp2600-wrapper.1in	2010-12-09 15:01:53.415806989 +0100
@@ -173,7 +173,7 @@ etc.  This option may appear more than o
 .TP
 .BI \-G\0 profile.icm
 Convert \fIprofile.icm\fP to a Postscript color rendering
-dictionary (CRD) using \fBfoo2zjs-icc2ps\fP and
+dictionary (CRD) using \fBicc2ps\fP and
 adjust the printer colors by using the Postscript \fBsetcolorrendering\fP
 operator.  If \fIprofile.icm\fP is none.icm, then prepare for ordering a
 ICM custom printer profile (i.e. from www.ICCFactory.com).
diff -Naurp foo2zjs/foo2hp2600-wrapper.in foo2zjs.oden/foo2hp2600-wrapper.in
--- foo2zjs/foo2hp2600-wrapper.in	2010-10-29 22:07:00.000000000 +0200
+++ foo2zjs.oden/foo2hp2600-wrapper.in	2010-12-09 15:01:53.416806919 +0100
@@ -619,7 +619,7 @@ create_crd() {
     #
     # Create a Postscript CRD
     #
-    ICC2PS=$PREFIX/bin/foo2zjs-icc2ps
+    ICC2PS=$PREFIX/bin/icc2ps
     if [ -x $ICC2PS ]; then
 	case "$GAMMAFILE" in
 	none.icm | */none.icm)
diff -Naurp foo2zjs/foo2lava-wrapper.1in foo2zjs.oden/foo2lava-wrapper.1in
--- foo2zjs/foo2lava-wrapper.1in	2010-03-02 22:50:49.000000000 +0100
+++ foo2zjs.oden/foo2lava-wrapper.1in	2010-12-09 15:01:53.416806919 +0100
@@ -167,7 +167,7 @@ etc.  This option may appear more than o
 .TP
 .BI \-G\0 profile.icm
 Convert \fIprofile.icm\fP to a Postscript color rendering
-dictionary (CRD) using \fBfoo2zjs-icc2ps\fP and
+dictionary (CRD) using \fBicc2ps\fP and
 adjust the printer colors by using the Postscript \fBsetcolorrendering\fP
 operator.  (WORK IN PROGRESS).
 .TP
diff -Naurp foo2zjs/foo2lava-wrapper.in foo2zjs.oden/foo2lava-wrapper.in
--- foo2zjs/foo2lava-wrapper.in	2010-10-29 22:07:00.000000000 +0200
+++ foo2zjs.oden/foo2lava-wrapper.in	2010-12-09 15:01:53.418807460 +0100
@@ -537,7 +537,7 @@ create_crd() {
     #
     # Create a Postscript CRD
     #
-    ICC2PS=$PREFIX/bin/foo2zjs-icc2ps
+    ICC2PS=$PREFIX/bin/icc2ps
     if [ -x $ICC2PS ]; then
         case "$GAMMAFILE" in
         none.icm | */none.icm)
diff -Naurp foo2zjs/foo2oak-wrapper.1in foo2zjs.oden/foo2oak-wrapper.1in
--- foo2zjs/foo2oak-wrapper.1in	2010-10-25 20:58:40.000000000 +0200
+++ foo2zjs.oden/foo2oak-wrapper.1in	2010-12-09 15:01:53.418807460 +0100
@@ -160,7 +160,7 @@ etc.  This option may appear more than o
 .TP
 .BI \-G\0 profile.icm
 Convert \fIprofile.icm\fP to a Postscript color rendering
-dictionary (CRD) using \fBfoo2zjs-icc2ps\fP and
+dictionary (CRD) using \fBicc2ps\fP and
 adjust the printer colors by using the Postscript \fBsetcolorrendering\fP
 operator.  (WORK IN PROGRESS).
 .TP
diff -Naurp foo2zjs/foo2oak-wrapper.in foo2zjs.oden/foo2oak-wrapper.in
--- foo2zjs/foo2oak-wrapper.in	2010-11-02 23:41:07.000000000 +0100
+++ foo2zjs.oden/foo2oak-wrapper.in	2010-12-09 15:01:53.419807444 +0100
@@ -546,7 +546,7 @@ create_crd() {
     #
     # Create a Postscript CRD
     #
-    ICC2PS=$PREFIX/bin/foo2zjs-icc2ps
+    ICC2PS=$PREFIX/bin/icc2ps
     if [ -x $ICC2PS ]; then
 	case "$GAMMAFILE" in
 	none | none.icm | */none.icm)
diff -Naurp foo2zjs/foo2qpdl-wrapper.1in foo2zjs.oden/foo2qpdl-wrapper.1in
--- foo2zjs/foo2qpdl-wrapper.1in	2010-03-29 19:11:36.000000000 +0200
+++ foo2zjs.oden/foo2qpdl-wrapper.1in	2010-12-09 15:01:53.419807444 +0100
@@ -164,7 +164,7 @@ etc.  This option may appear more than o
 .TP
 .BI \-G\0 profile.icm
 Convert \fIprofile.icm\fP to a Postscript color rendering
-dictionary (CRD) using \fBfoo2zjs-icc2ps\fP and
+dictionary (CRD) using \fBicc2ps\fP and
 adjust the printer colors by using the Postscript \fBsetcolorrendering\fP
 operator.  (WORK IN PROGRESS).
 .TP
diff -Naurp foo2zjs/foo2qpdl-wrapper.in foo2zjs.oden/foo2qpdl-wrapper.in
--- foo2zjs/foo2qpdl-wrapper.in	2010-10-29 22:07:00.000000000 +0200
+++ foo2zjs.oden/foo2qpdl-wrapper.in	2010-12-09 15:01:53.420807433 +0100
@@ -518,7 +518,7 @@ create_crd() {
     #
     # Create a Postscript CRD
     #
-    ICC2PS=$PREFIX/bin/foo2zjs-icc2ps
+    ICC2PS=$PREFIX/bin/icc2ps
     if [ -x $ICC2PS ]; then
 	case "$GAMMAFILE" in
 	none | none.icm | */none.icm)
diff -Naurp foo2zjs/foo2slx-wrapper.1in foo2zjs.oden/foo2slx-wrapper.1in
--- foo2zjs/foo2slx-wrapper.1in	2007-12-20 07:22:36.000000000 +0100
+++ foo2zjs.oden/foo2slx-wrapper.1in	2010-12-09 15:02:16.223838826 +0100
@@ -129,7 +129,7 @@ etc.  This option may appear more than o
 .TP
 .BI \-G\0 profile.icm
 Convert \fIprofile.icm\fP to a Postscript color rendering
-dictionary (CRD) using \fBfoo2zjs-icc2ps\fP and
+dictionary (CRD) using \fBicc2ps\fP and
 adjust the printer colors by using the Postscript \fBsetcolorrendering\fP
 operator.  (WORK IN PROGRESS).
 .TP
diff -Naurp foo2zjs/foo2slx-wrapper.in foo2zjs.oden/foo2slx-wrapper.in
--- foo2zjs/foo2slx-wrapper.in	2010-10-29 22:07:00.000000000 +0200
+++ foo2zjs.oden/foo2slx-wrapper.in	2010-12-09 15:01:53.420807433 +0100
@@ -503,7 +503,7 @@ create_crd() {
     #
     # Create a Postscript CRD
     #
-    ICC2PS=$PREFIX/bin/foo2zjs-icc2ps
+    ICC2PS=$PREFIX/bin/icc2ps
     if [ -x $ICC2PS ]; then
 	case "$GAMMAFILE" in
 	none | none.icm | */none.icm)
diff -Naurp foo2zjs/foo2xqx-wrapper.in foo2zjs.oden/foo2xqx-wrapper.in
--- foo2zjs/foo2xqx-wrapper.in	2010-10-29 22:07:00.000000000 +0200
+++ foo2zjs.oden/foo2xqx-wrapper.in	2010-12-09 15:01:53.421870534 +0100
@@ -499,7 +499,7 @@ create_crd() {
     #
     # Create a Postscript CRD
     #
-    ICC2PS=$PREFIX/bin/foo2zjs-icc2ps
+    ICC2PS=$PREFIX/bin/icc2ps
     if [ -x $ICC2PS ]; then
 	if [ -x /usr/bin/logger ]; then
 	    logger -t "$BASENAME" -p lpr.info -- \
diff -Naurp foo2zjs/foo2zjs-wrapper.1in foo2zjs.oden/foo2zjs-wrapper.1in
--- foo2zjs/foo2zjs-wrapper.1in	2010-12-08 23:36:04.000000000 +0100
+++ foo2zjs.oden/foo2zjs-wrapper.1in	2010-12-09 15:01:53.421870534 +0100
@@ -202,7 +202,7 @@ etc.  This option may appear more than o
 .TP
 .BI \-G\0 profile.icm
 Convert \fIprofile.icm\fP to a Postscript color rendering
-dictionary (CRD) using \fBfoo2zjs-icc2ps\fP and
+dictionary (CRD) using \fBicc2ps\fP and
 adjust the printer colors by using the Postscript \fBsetcolorrendering\fP
 operator.  (WORK IN PROGRESS).
 .TP
diff -Naurp foo2zjs/foo2zjs-wrapper.in foo2zjs.oden/foo2zjs-wrapper.in
--- foo2zjs/foo2zjs-wrapper.in	2010-12-08 23:21:51.000000000 +0100
+++ foo2zjs.oden/foo2zjs-wrapper.in	2010-12-09 15:01:53.422870452 +0100
@@ -669,7 +669,7 @@ create_crd() {
     #
     # Create a Postscript CRD
     #
-    ICC2PS=$PREFIX/bin/foo2zjs-icc2ps
+    ICC2PS=$PREFIX/bin/icc2ps
     if [ -x $ICC2PS ]; then
 	case "$GAMMAFILE" in
 	none | none.icm | */none.icm)
diff -Naurp foo2zjs/Makefile foo2zjs.oden/Makefile
--- foo2zjs/Makefile	2010-12-08 21:42:15.000000000 +0100
+++ foo2zjs.oden/Makefile	2010-12-09 15:03:03.186807724 +0100
@@ -184,13 +184,6 @@ FILES	=	\
 		foomatic-db/*/*.xml \
 		foomatic-test \
 		getweb.in \
-		icc2ps/*.[ch] \
-		icc2ps/*.1in \
-		icc2ps/Makefile \
-		icc2ps/AUTHORS \
-		icc2ps/COPYING \
-		icc2ps/README \
-		icc2ps/README.foo2zjs \
 		ppd-adjust \
 		PPD/*.ppd \
 		crd/zjs/*.crd \
@@ -356,7 +349,7 @@ JBGOPTS=-m 16 -d 0 -p 92	# Equivalent op
 #
 # The usual build rules
 #
-all:	all-test $(PROGS) $(BINPROGS) $(SHELLS) getweb all-icc2ps man doc \
+all:	all-test $(PROGS) $(BINPROGS) $(SHELLS) getweb man doc \
 	all-done
 
 all-test:
@@ -494,9 +487,6 @@ getweb: getweb.in Makefile
 	    -e 's@^PREFIX=.*@PREFIX=$(PREFIX)@' || (rm -f $@ && exit 1)
 	chmod 555 $@
 
-all-icc2ps:
-	cd icc2ps; $(MAKE) all
-
 ok: ok.o $(LIBJBG)
 	$(CC) $(CFLAGS) ok.o $(LIBJBG) -o $@
 
@@ -539,7 +529,7 @@ command2foo2lava-pjl.o: command2foo2lava
 #
 # Installation rules
 #
-install: all install-test install-prog install-icc2ps install-extra \
+install: all install-test install-prog install-extra \
 	    install-crd install-foo install-ppd \
 	    install-gui install-desktop install-filter \
 	    install-man install-doc
@@ -648,12 +638,6 @@ install-foo:
 	    foomatic-combo-xml -O >overview.xml; \
 	fi
 
-install-icc2ps:
-	#
-	# Install ICM to Postscript file conversion utility
-	#
-	cd icc2ps; $(MAKE) PREFIX=$(PREFIX) install
-
 install-crd:
 	#
 	# Install prebuilt CRD files (from m2300w project)
@@ -953,7 +937,6 @@ uninstall:
 	-rm -f $(MANDIR)/man1/foo2hiperc*.1 $(MANDIR)/man1/hipercdecode.1
 	-rm -f $(MANDIR)/man1/gipddecode.1
 	-rm -f $(MANDIR)/man1/arm2hpdl.1 $(MANDIR)/man1/usb_printerid.1
-	-rm -f $(MANDIR)/man1/foo2zjs-icc2ps.1
 	-rm -rf /usr/share/foo2zjs/
 	-rm -rf /usr/share/foo2hp/
 	-rm -rf /usr/share/foo2oak/
@@ -975,7 +958,6 @@ uninstall:
 	-rm -f /usr/bin/gipddecode
 	-rm -f /usr/bin/opldecode
 	-rm -f /usr/bin/rodecode
-	-rm -f /usr/bin/foo2zjs-icc2ps
 	-rm -f /usr/bin/foo2zjs-pstops
 	-rm -f /usr/bin/command2foo2lava-pjl
 	-rm -f /usr/lib/cups/filter/command2foo2lava-pjl
@@ -1017,7 +999,6 @@ clean:
 	-rm -f *.zjs *.zm *.zc *.zc? *.zc?? *.oak *.pbm *.pksm *.cmyk
 	-rm -f pksm2bitcmyk
 	-rm -f *.icm.*.ps
-	cd icc2ps; $(MAKE) $@
 
 #
 # Header dependencies
@@ -1165,7 +1146,7 @@ pprtest-3.oak: FRC
 #
 #	icc2ps regression tests
 #
-ICC2PS=./icc2ps/foo2zjs-icc2ps
+ICC2PS=/usr/bin/icc2ps
 icctest:
 	for g in *.icm; do \
 	    for i in 0 1 2 3; do \
@@ -1332,9 +1313,9 @@ install-doc: doc
 
 GROFF=/usr/local/test/bin/groff
 GROFF=groff
-manual.pdf: $(MANPAGES) icc2ps/foo2zjs-icc2ps.1
+manual.pdf: $(MANPAGES)
 	-$(GROFF) -t -man \
-	    `ls $(MANPAGES) icc2ps/foo2zjs-icc2ps.1 | sort` \
+	    `ls $(MANPAGES) | sort` \
 	    | ps2pdf - $@
 
 README: README.in
