diff -up foo2zjs/Makefile.0001~ foo2zjs/Makefile
--- foo2zjs/Makefile.0001~	2018-11-27 18:44:15.456790489 +0100
+++ foo2zjs/Makefile	2018-11-27 18:45:27.953228963 +0100
@@ -1061,10 +1061,10 @@ install-hotplug-osx:
 	cd osx-hotplug; $(MAKE) PREFIX=$(PREFIX) install-hotplug
 
 install-filter:
-	if [ "$(CUPS_SERVERBIN)" != "" ]; then \
-	    $(INSTALL) -d $(CUPS_SERVERBIN)/filter; \
-	    ln -sf $(BIN)/command2foo2lava-pjl $(CUPS_SERVERBIN)/filter/; \
-	fi
+	#if [ "$(CUPS_SERVERBIN)" != "" ]; then \
+	#    $(INSTALL) -d $(CUPS_SERVERBIN)/filter; \
+	#    ln -sf $(BIN)/command2foo2lava-pjl $(CUPS_SERVERBIN)/filter/; \
+	#fi
 
 CUPSDCONF=/etc/cups/cupsd.conf
 CUPSFILESCONF=/etc/cups/cups-files.conf
